app/src/main/java/com/duckduckgo/app/httpsupgrade/BloomFilter.kt

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2018 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.httpsupgrade
+
+class BloomFilter {
+
+    private val nativePointer: Long
+
+    init {
+        System.loadLibrary("https-bloom-lib")
+    }
+
+    constructor(maxItems: Int, targetProbability: Double) {
```

Comment:
It is an appropriate part of the API for this class and matching the C++ code, so I don't feel a need to hide it. I'm happy to if you feel strongly about it.

Reply:
Just because it's in the C++ code, does that mean we need to have it exposed in Kotlin? If yes, then happy to leave it.

But if not, it seems we only call it from testing and it might not exist otherwise, hence why I was thinking that annotation might help explain its existence.

But i don't feel strongly about it.